experimental/framework/tinylicious-client/src/TinyliciousAudience.ts

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ServiceAudience } from "@fluid-experimental/fluid-static";
+import { IClient } from "@fluidframework/protocol-definitions";
+import { ITinyliciousAudience, TinyliciousLastEditedResult, TinyliciousMember } from "./interfaces";
+
+export class TinyliciousAudience extends ServiceAudience implements ITinyliciousAudience {
+  /**
+   * @inheritdoc
+   */
+  public getMembers(): Map<string, TinyliciousMember> {
```

Comment:
Is Map a good fit for usage scenarios here?  No suggestion to change at the moment, just thinking that it sort of assumes the user will want to retrieve by user ID.

Like the demo usage you added in external-controller looks like it would be just as happy with an array, for example.

Reply:
Right, I was thinking that the user may have other places where they are storing values tagged with the user ID. If they ever need to correlate that with say the clients returned a specific user, they could do a simple look-up on this map. And if they want to use it as an array, they can just do `Array.from(getMembers().values())`.
I can convert it to an array but when we are preparing the response, we need to initially build a Map anyways to correlate multiple clients for the same user. We would just be moving the `Array.from` call internally instead of reducing the processing required